client/state/data-layer/wpcom/activity-log/activity/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { ACTIVITY_LOG_REQUEST } from 'state/action-types';
+import {
+	activityLogError,
+	activityLogUpdate,
+} from 'state/activity-log/actions';
+// import addQueryArgs from 'lib/route/add-query-args';
+
+const activityLogRequest = ( { dispatch }, action ) => {
+	dispatch( http( {
+		apiVersion: '1',
+		method: 'GET',
+		path: `/sites/${ action.siteId }/activity`,
```

Comment:
The convention of the folder structure of data layer modules is to match the endpoint URL, so we should move this up to `/wpcom/activity` I think.